src/index.tsx

```diff
@@ -1,7 +1,8 @@
 import ReactDOM from 'react-dom';
-import App from './App';
+import { App } from './App';
+import { names } from './api/clockNames';
 
 ReactDOM.render(
-  <App />,
+  <App listOfNames={names} />,
```

Comment:
```suggestion
  <App />,
```

Better not to give props directly to the `App` component.
Import your `listOfNames` inside the App

Reply:
got it